experimental/CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ All notable changes to experimental packages in this project will be documented
 * docs(instrumentation): better docs for supportedVersions option [#4693](https://github.com/open-telemetry/opentelemetry-js/pull/4693) @blumamir
 * docs: align all supported versions to a common format [#4696](https://github.com/open-telemetry/opentelemetry-js/pull/4696) @blumamir
 * refactor(examples): use new exported string constants for semconv in experimental/examples/opencensus-shim [#4763](https://github.com/open-telemetry/opentelemetry-js/pull/4763#pull) @Zen-cronic
+* docs(opentelemetry-browser-detector): Use tree-shakeable string constants for semconv in Readme [#4768](https://github.com/open-telemetry/opentelemetry-js/pull/4768) @JohannesHuster
```

Comment:
Sorry for the churn, this will need to move up into the Unreleased section now

Reply:
@JamieDanielson No worries, always good to see a release! I added the change as a new commit for now, to make reviewing easier, but I'm happy to rebase and remove some unnecessary commits before merging.

Reply:
Failing tests are already being addressed here, I think: https://github.com/open-telemetry/opentelemetry-js/pull/4777

Reply:
The test fix was merged so I've merged main here. Hopefully that passes now 🤞

Reply:
Also just saw your other comment, no rebase is necessary 🙂 . Rebases can make it difficult to see what changes have occurred during review.

Reply:
Perfect, thanks 🙂 Tests are looking good now 😌